emission/tests/storageTests/TestTokenQueries.py

```diff
@@ -73,21 +76,73 @@ def test_auth_fail_file_insert(self):
             tlm = enat.TokenListMethod()
             tlm.verifyUserToken('z')
 
-    def test_run_script(self):
-        os.system("python3 bin/auth/insert_tokens.py emission/tests/storageTests/token_test_files/tokens.txt")
+    def test_run_script_file(self):
+        os.system("python3 bin/auth/insert_tokens.py --file emission/tests/storageTests/token_test_files/tokens.txt")
         self.assertEqual(esdt.get_all_tokens(), ['a','b','c','d'])
 
-    def test_run_script_auth(self):
-        os.system("python3 bin/auth/insert_tokens.py emission/tests/storageTests/token_test_files/tokens.txt")
+    def test_run_script_file_auth(self):
```

Comment:
what is the difference between this and `test_run_script_file`? you might want to add some comments explaining this...

Reply:
It's the same, I just renamed it to specify what argument it is testing since I added new options.